pkgs/applications/graphics/fiji/default.nix

```
@@ -0,0 +1,80 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, autoPatchelfHook
+, jdk11
+, makeDesktopItem
+, runtimeShell
+}:
+let
+  pname = "fiji";
+  version = "20201104-1356";
+
+  niceName = "Fiji Is Just ImageJ";
+
+  desktopItem = makeDesktopItem {
+    name = "fiji";
+    exec = "fiji %F";
+    icon = "fiji";
+    mimeType = "image/*;";
+    comment = "Scientific Image Analysis";
+    desktopName = niceName;
+    genericName = niceName;
+    categories = "Education;Science;ImageProcessing;";
+    terminal = false;
+    startupNotify = true;
+    extraEntries = ''
+      Version=1.0
+      TryExec=fiji
+      X-GNOME-FullName=${niceName}
+      StartupWMClass=fiji-Main
+    '';
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version;
```

```suggestion
stdenv.mkDerivation rec {
  pname = "fiji";
  version = "20201104-1356";
```
and the let in can be removed.